content/kubermatic/v2.25/tutorials-howtos/kkp-autoscaler/_index.en.md

```diff
@@ -187,7 +187,16 @@ You can delete autoscaler from where you edit it above and select delete.
 ![Delete Autoscaler](/img/kubermatic/v2.25/tutorials/cluster-autoscaler/delete-autoscaler.png?classes=shadow,border "Delete Autoscaler")
 
 
- Once it has been deleted, you can check the cluster to ensure that the autoscaler has been deleted using the command `kubectl get pods -n kube-system`.
+ Once it has been deleted, you can check the cluster to ensure that the cluster autoscaler has been deleted using the command `kubectl get pods -n kube-system`.
+
+
+## Customize KKP Autoscaler
+
+You can customize the cluster autoscaler addon in order to override the cluster autoscaler deployment definition to set or pass the required flag(s) by following the instructions provided over [here](https://docs.kubermatic.com/kubermatic/v2.25/architecture/concept/kkp-concepts/addons/#custom-addons), if you are facing any of the below mentioned issues as described in the [cluster autoscaler FAQs](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md):
```

Comment:
What about something like this:
```suggestion
You can customize the cluster autoscaler addon in order to override the cluster autoscaler deployment definition to set or pass the required flag(s) by following the instructions provided [in the Addons document](https://docs.kubermatic.com/kubermatic/v2.25/architecture/concept/kkp-concepts/addons/#custom-addons).

### Suggestions and Known Issues

Some of [the know issues in the cluster-autoscaler](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md) can be mitigated by adjusting the cluster-autoscaler deployment to pass additional flags, for example:
```

Reply:
Also please use `ref` to access other KKP documents, such as `{{< ref "../../architecture/concept/kkp-concepts/addons/#custom-addons" >}}` (I didn't test this URL, please make sure to test it and fix it if needed).

Reply:
done!